opennms-doc/guide-install/src/asciidoc/text/minion/configure-minion.adoc

```diff
@@ -30,6 +30,15 @@ admin@minion()> scv:set opennms.http minion minion
 admin@minion()> scv:set opennms.broker minion minion
 ----
 
+Another way to configure credentials is to use the `scvcli` utility in your _Minion_ `bin` directory.
+
+.Configure the credentials by using the command line utility `scvcli`
+[source]
+----
+[root@localhost /usr/share/minion]# $ ./bin/scvcli set opennms.http minion minion
```

Comment:
I don't see `/usr/share/minion` being referenced anywhere else. What example are you referring to exactly?

The ones I do see connect via the Karaf shell, where the working directory doesn't matter.